test/StatsTestCases.test.js

```diff
@@ -52,7 +52,7 @@ describe("StatsTestCases", () => {
 				if(/error$/.test(testName)) {
 					stats.hasErrors().should.be.equal(true);
 				} else if(stats.hasErrors()) {
-					done(new Error(stats.toJson().errors.join("\n\n")));
+					return done(new Error(stats.toJson().errors.join("\n\n")));
```

Comment:
I was constantly getting a confusing `Error: done() called multiple times` while working on these tests.  I'm pretty sure we want to return here, but please correct me if I'm wrong.